src/components/WebexSettings/Select.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Select component
+ *
+ * @param {object} props Data passed to the component
+ * @param {object[]} props.options Array of HMTL option elements
+ * @param {string} props.className  Custom CSS class to apply
+ *
+ * @returns {object} JSX of the element
+ */
+export default function Select({className, options, ...props}) {
+  const mainClasses = {
+    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-select`]: true,
+    [className]: !!className,
+  };
+
+  return (
+    <div className={classNames(mainClasses)}>
```

Comment:
if not 100% needed, please remove this `<div>` wrapper and put the class directly on the `<select>` element.